app/ts/utils/errors.ts

```diff
@@ -28,7 +28,7 @@ export const isNewBlockAbort = (error: Error) => error.message.includes(NEW_BLOC
 
 export function printError(error: unknown) {
 	if (error instanceof Error && 'data' in error) {
-		return console.error(`Error: ${ error.message }\n${ JSON.stringify(error.data) }`)
+		return console.error(`Error: ${ error.message }\n${ JSON.stringify(error.data) }\n${ error.stack !== undefined ? error.stack : ''}`)
```

Comment:
good catch, but being object is not enough, it can still contain that kind of stuff. I think I wrap that to try catch instead as there's no safe way to check that